travis_configure.sh

```diff
@@ -129,7 +129,8 @@ run_tests()
             make deb
         elif [[ "$DIST_PACKAGE_TARGET" == "RPM" ]]; then
             docker run -d --name db_container centos7_mysql56
-            docker exec -u root db_container sh -c \
+            docker exec -e MYSQL_USER=root -e MYSQL_PASSWORD= \
+                -u root db_container sh -c \
```

Comment:
Can we not read these values from somewhere else in the config rather than hard coding it?

Reply:
Yeah, true, they are defined as env variables in the travis build environment already. So we can read them from there